app/src/main/java/com/example/gradeassist/ui/quiz_results/QuizResultsFragment.kt

```diff
@@ -28,4 +30,11 @@ class QuizResultsFragment : Fragment()  {
         })
         return root
     }
+
+    override fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
+        super.onActivityResult(requestCode, resultCode, data)
+        if (requestCode == Activity.RESULT_OK){
+            //TODO Add the needed onActivityResult if statements with resultCodes needed for this fragments
```

Comment:
Ditto comment above.  Since this function is not used, it should be removed, and only added when needed.  

It's a good idea to keep a codebase as small as possible to accomplish a given task.  Any code that is there, and not being used, is a distraction to a developer trying to understand the codebase.